app/models/developers/title.rb

```diff
@@ -0,0 +1,45 @@
+module Developers
+  class Title
+    private attr_reader :query
+
+    def initialize(query)
+      @query = query
+    end
+
+    def title
+      components = []
+      components << "Hire"
+      components << role_level if role_level
+      components << "freelance" if freelance?
+      components << "Ruby on Rails developers"
+      components << "in #{country}" if country
+      components.join(" ")
+    end
+
+    private
+
+    def role_level
+      if query.role_levels.one?
+        if query.role_levels.first == :mid
+          "mid-level"
+        elsif query.role_levels.first == :c_level
+          "C-level"
+        else
+          query.role_levels.first
+        end
+      end
+    end
```

Comment:
I feel like I should be able to get these from the translations file but can't figure out how.

Reply:
You can use a fallback for this:

Reply:
Hmm, is there any where I can use the existing `activerecord.attributes.role_level` keys?

Reply:
Yeah, just adjust the scope: `[:activerecord, :attributes, :role_level]`